packages/apps/shopify-api/lib/billing/types.ts

```diff
@@ -121,7 +121,9 @@ export type BillingConfigItem<
 > =
   FeatureEnabled<Future, 'v10_lineItemBilling'> extends true
```

Comment:
I attempted to create a type that would allow you only to set `v10_lineItemBilling` or `v11_lineItemBilling` but things got quite complicated. So taking suggestions on if/how we want to do that if we deem necessary.